src/movies.js

```diff
@@ -1,22 +1,108 @@
+const movies = require("./data");
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors() {}
+function getAllDirectors(obj) {
```

Comment:
Hola Pool el metodo map retorna un nuevo array puedes guardarlo en una variable y retornarla o simplemente retornar el array que da el map, por eso no es necesario hacer push como le estabamos haciendo con for o forEach 
```suggestion
function getAllDirectors(obj) {
  return obj.map((obj)=>obj.director)
}
getAllDirectors(movies);
```